libs/template/templates/default-sql/template/{{.project_name}}/README.md.tmpl

```diff
@@ -0,0 +1,41 @@
+# {{.project_name}}
+
+The '{{.project_name}}' project was generated by using the default-sql template.
+
+## Getting started
+
+1. Install the Databricks CLI from https://docs.databricks.com/dev-tools/cli/databricks-cli.html
```

Comment:
A more direct URL would be https://docs.databricks.com/en/dev-tools/cli/install.html

Reply:
```suggestion
1. Install the Databricks CLI from https://docs.databricks.com/dev-tools/cli/install.html

```